### PR TITLE
File upload onDrop

### PR DESCRIPTION
**Pull request by tsumo**
## PR includes

- [x] Feature
- [x] DevOps

### CODE REVIEW

File: src/harmony/FileUpload/FileUpload.tsx [hunk 1]
```diff
@@ -228,7 +228,7 @@ export const FileUpload = forwardRef<HTMLInputElement, FileUploadProps>(
                 </div>
 
                 {nullable(uploadedFiles, (fileList) => (
-                    <div className={styles.FileUpload_Filelist}>
```
Comment (9teen90nine) [inline src/harmony/FileUpload/FileUpload.tsx:231]: why did you remove a style?

Comment (tsumo) [reply src/harmony/FileUpload/FileUpload.tsx:231]: css file doesn't have this class

**Review by LamaEats — APPROVED** [review]
